.travis.yml

```diff
@@ -4,18 +4,35 @@ arch:
   - ppc64le
 
 rvm:
-  - 2.5.0
-  - 2.5.8
-  - 2.6.0
-  - 2.6.10
-  - 2.7.0
-  - 2.7.6
-  - 3.0.1
+  - 2.7.7
```

Comment:
We need to keep these older Ruby versions. I want to ensure that we are testing in the versions that would be used in Rails 3 projects.

Reply:
> I want to ensure that we are testing in the versions that would be used in Rails 3 projects.

OK